core/src/test/scala/org/broadinstitute/dsde/rawls/user/UserServiceSpec.scala

```diff
@@ -1279,36 +1303,93 @@ class UserServiceSpec
   it should "map azure data for a BPM project" in {
     val billingProfile = new ProfileModel().id(UUID.randomUUID()).cloudPlatform(BPMCloudPlatform.AZURE)
     val projectName = RawlsBillingProjectName(UUID.randomUUID().toString)
+    val landingZoneId = UUID.randomUUID()
+    val lzRegion = "dummy-region"
     val project = RawlsBillingProject(
       projectName,
       CreationStatuses.Ready,
       None,
       None,
-      billingProfileId = Some(billingProfile.getId.toString)
+      billingProfileId = Some(billingProfile.getId.toString),
+      landingZoneId = Some(landingZoneId.toString)
     )
 
     val repository = mock[BillingRepository]
     when(repository.getBillingProjects(ArgumentMatchers.eq(Set(projectName))))
       .thenReturn(Future.successful(Seq(project)))
 
-    val noRole = SamRolesAndActions(Set.empty, Set.empty)
-    val creatorRole =
-      SamRolesAndActions(Set(SamBillingProjectRoles.workspaceCreator), Set(SamBillingProjectActions.createWorkspace))
     val billingResource = SamUserResource(projectName.value, creatorRole, noRole, noRole, Set.empty, Set.empty)
 
     val samDAO = mock[SamDAO](RETURNS_SMART_NULLS)
     when(samDAO.listUserResources(SamResourceTypeNames.billingProject, testContext))
       .thenReturn(Future.successful(Seq(billingResource)))
     val bpmDAO = mock[BillingProfileManagerDAO](RETURNS_SMART_NULLS)
     when(bpmDAO.getAllBillingProfiles(testContext)).thenReturn(Future.successful(Seq(billingProfile)))
-    val userService = getUserService(samDAO = samDAO, bpmDAO = bpmDAO, billingRepository = Some(repository))
+
+    val wsmDAO = mock[WorkspaceManagerDAO](RETURNS_SMART_NULLS)
+    when(wsmDAO.getLandingZone(landingZoneId, testContext)).thenReturn(new AzureLandingZone().region(lzRegion))
+
+    val userService = getUserService(samDAO = samDAO,
+                                     bpmDAO = bpmDAO,
+                                     billingRepository = Some(repository),
+                                     workspaceManagerDao = wsmDAO
+    )
+
+    val expected = Seq(
+      RawlsBillingProjectResponse(
+        Set(ProjectRoles.User),
+        project.copy(azureManagedAppCoordinates = Some(AzureManagedAppCoordinates(null, null, null))),
+        CloudPlatform.AZURE,
+        Option(false),
+        Option(lzRegion)
+      )
+    )
+
+    Await.result(userService.listBillingProjectsV2(), Duration.Inf) should contain theSameElementsAs expected
+  }
+
+  it should "map handle the landing zone being missing in BPM project" in {
```

Comment:
Nit: I understand what this means since I know the scope of changes in this PR , but the wording is kind of confusing otherwise.

Reply:
Thanks, it was a copy/paste error.